ios/brave-ios/Sources/Onboarding/Callouts/OnboardingTranslateView.swift

```diff
@@ -0,0 +1,121 @@
+// Copyright 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import BraveShared
+import BraveUI
+import DesignSystem
+import Foundation
+import Strings
+import SwiftUI
+
+public struct OnboardingTranslateView: View {
+  @Environment(\.dismiss) private var dismiss
+
+  private var onContinueButtonPressed: (() -> Void)?
+  private var onDisableFeature: (() -> Void)?
+
+  public init(onContinueButtonPressed: (() -> Void)? = nil, onDisableFeature: (() -> Void)? = nil) {
+    self.onContinueButtonPressed = onContinueButtonPressed
+    self.onDisableFeature = onDisableFeature
+  }
+
+  public var body: some View {
+    ScrollView(.vertical) {
+      VStack(spacing: 0.0) {
+        Image(
+          uiImage: UIImage(named: "translate-onboarding-icon", in: .module, compatibleWith: nil)!
+        )
```

Comment:
No need to drop to `UIImage`, you can just do `Image("translate-onboarding-icon", bundle: .module)`